include/osqp_api_constants.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 /*************************
 * Linear System Solvers *
 *************************/
-enum linsys_solver_type { QDLDL_SOLVER, MKL_PARDISO_SOLVER, CUDA_PCG_SOLVER };
+enum linsys_solver_type { QDLDL_SOLVER, MKL_PARDISO_SOLVER, CUDA_PCG_SOLVER, UNKNOWN_SOLVER=99 };
```

Comment:
Due to C++ (catch2) complaining about mixing int/enum types. Already approved in master.